cmd/editorconfig-checker/main.go

```diff
@@ -60,6 +60,9 @@ func init() {
 		configPaths = append(configPaths, defaultConfigFileNames[:]...)
 	} else {
 		configPaths = append(configPaths, configFilePath)
+		if configFilePath == ".ecrc" {
```

Comment:
I don't think the if statement is at the right place:

- here you are testing the content parameter `--config`
- it could be a relative or full path, so testing with an equality looks insufficient
- if someone passed a config path, even if it's the old name, we should accept silently.

Reply:
Here is a suggestion

https://github.com/editorconfig-checker/editorconfig-checker/pull/389#discussion_r1797617246

Reply:
@ccoVeille I have updated the code accordingly.